docs/docsite/rst/playbooks_filters.rst

```diff
@@ -609,10 +609,25 @@ override those in `b`, and so on.
 This behaviour does not depend on the value of the `hash_behaviour`
 setting in `ansible.cfg`.
 
-.. _extract_filter:
+.. _extract_filters:
 
 Extracting values from containers
 `````````````````````````````````
+.. versionadded:: 2.5
+
+The `slice` filter can be used to extract the values of specific keys from a
+hash::
+
+    {{ {'x': 1, 'y': 2, 'z': 3 } | slice(['x', 'z']) }}
+
+This will result in::
+
+    [1, 2]
+
+The filter is especially useful when you have a list of hashes and want to
+return the value of a key common to all hashes::
+
+    {{ variable_registered_in_loop.results | map('slice', 'stdout') | list }}
```

Comment:
you actually want to use map's `attribute` in this case, which will return a list , this returns a list of lists (1 per element)

Reply:
I should have studied better.